### ISSUE TITLE
conf loss question!!

### ISSUE THREAD

**Issue by Jiachenyin1** [Tianxiaomo/pytorch-YOLOv4#222]
pred_ious = bboxes_iou(pred[b].view(-1, 4), truth_box, xyxy=False)
            pred_best_iou, _ = pred_ious.max(dim=1)
            pred_best_iou = (pred_best_iou > self.ignore_thre)
            pred_best_iou = pred_best_iou.view(pred[b].shape[:3])
            # set mask to zero (ignore) if pred matches truth

            obj_mask[b] = ~ pred_best_iou

能问下obj_mask 应该是论文的ignore 吧 ， 如 gt的iou 超过 设定值 ， 则假设为0 ， 但是我没有太懂 的是 back_ground 这里直接设置的是 1 ， 然后：
            loss_obj += F.binary_cross_entropy(input=output[..., 4], target=target[..., 4], size_average=False)
这里合理吗？
求解释？
